### DISCUSSION TITLE
[  DEBUG  ]  Checking /var/mail permissions

### DISCUSSION THREAD

**Discussion by ckvist72** [docker-mailserver/docker-mailserver#3096] [Q&A]
On start i get stuck here: [  DEBUG  ]  Checking /var/mail permissions

**Comment by polarathene** [comment]
You'd need to provide more information, like your configuration.

Are you using LDAP? Are you running this on macOS or Windows as the Docker host? Are you using a rootless container?

Usually problems like that are due to any of those, or someone who is new to Linux and/or Docker.
